src/SSL/ValidationClient.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace UKFast\SDK\SSL;
+
+use UKFast\SDK\Client as BaseClient;
+use UKFast\SDK\SSL\Entities\ValidationResult;
+
+class ValidationClient extends BaseClient
+{
+    protected $basePath = 'ssl/';
+
+    /**
+     * Validation Result API fields which need mapping
+     *
+     * @var array
+     */
+    public $validationMap = [
+        'alt_hosts' => 'altHosts',
+        'expiry_timestamp' => 'expiryTimestamp',
+    ];
+
+    /**
+     * Validate a certificate against its key or key and CA bundle
+     *
+     * @param string $certificate
+     * @param string $certificateKey
+     * @param string $caBundle
+     * @return ValidationResult
+     */
+    public function validate($certificate, $certificateKey, $caBundle)
```

Comment:
Should this be optional to cater for self signed certificates without having to pass a third parameter?